hermes/file_gen.go

```diff
@@ -1,40 +1,33 @@
 package hermes
 
-
 import (
 	"fmt"
-	"strconv"
 	"crypto/sha1"
 )
 
 type HermesFile struct {
 	name string
 	contents string
 }
-func (file *HermesFile) GenerateFileName(file_ID int, file_checksum string) {
-	id_string := ""
-	if file_ID < 10 {
-		id_string = "0" + strconv.Itoa(file_ID)
-	} else {
-		id_string = strconv.Itoa(file_ID)
-	}
-	file.name := "Hermes_"+id_string+file_checksum
+func (file *HermesFile) generateFileName(file_ID int, file_checksum string) {
+	file_name :=  fmt.Sprintf("Hermes_%02d%v", file_ID, file_checksum)
+	file.name = file_name
 }
-
 func (file *HermesFile) generateFileContents(file_ID int) {
 	file.contents = "jhfvjhdfjhfjjhjhdfvjvcvfjh";
 }
-
-func (file HermesFile) GenerateFileChecksum() string {
+func (file HermesFile) generateFileChecksum() string {
+	
 	file_contents := []byte(file.contents)
 	hash := sha1.Sum(file_contents)
-	checksum := fmt.Sprintf("_%x", hash)
+	checksum := fmt.Sprintf("%s%x","_", hash)
 	return checksum;
```

Comment:
How about we return just the checksum and prepend the `_` inside generateFileName()?
```suggestion
        // return checksum in hex notation
	return fmt.Sprintf("%x", hash)
```